recipes/pybind11/all/conanfile.py

```diff
@@ -31,6 +31,7 @@ def source(self):
 
     def generate(self):
         tc = CMakeToolchain(self)
+        tc.variables["PYBIND11_FINDPYTHON"] = True
```

Comment:
@tttapa Why do you need `PYBIND11_FINDPYTHON` only now? As far as I see, it's active by default: https://github.com/pybind/pybind11/blob/v3.0.0/CMakeLists.txt#L86. What error are you trying to mitigate?